core/services/keystore/keys/ocr2key/offchain_keyring_test.go

```diff
@@ -16,3 +18,31 @@ func TestOffchainKeyring(t *testing.T) {
 	pubKey := kr.OffchainPublicKey()
 	assert.True(t, bytes.Equal(kr.signingKey.Public().(ed25519.PublicKey), pubKey[:]))
 }
+
+func TestOffchainKeyring_Decrypt(t *testing.T) {
+	kr, err := newOffchainKeyring(cryptorand.Reader, cryptorand.Reader)
+	require.NoError(t, err)
+
+	originalMessage := []byte("test")
+
+	encryptedMessage, err := Encrypt(kr.ConfigEncryptionPublicKey(), originalMessage)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	decryptedMessage, err := kr.Decrypt(encryptedMessage)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	assert.Equal(t, originalMessage, decryptedMessage)
+}
+
+func Encrypt(peerPublicKey [curve25519.PointSize]byte, plaintext []byte) (ciphertext []byte, err error) {
```

Comment:
if that's a test helper pass `t *testing.T` as the first arg, call `t.Helper()` at the beginning of the func, and for every non-nil err call `t.Fatalf()`.